eva-tools/src/test/java/embl/ebi/variation/eva/vcfdump/VariantExporterTest.java

```diff
@@ -84,190 +88,202 @@ public static void setUpClass() throws IOException, InterruptedException, URISyn
+        QueryOptions query = new QueryOptions();
+        Map<String, List<VariantContext>> exportedVariants = exportAndCheck(query, studies, region);
+        checkExportedVariants(query, studies, exportedVariants);
     }
 
     @Test
-    public void testFilterUsingIntersectingRegions() throws Exception {
-        List<String> files = Arrays.asList("5");
-        List<String> studies = Arrays.asList("7");
-        QueryOptions query = getQuery(files, studies);
-        String outputDir = "/tmp/";
-
-        // tell all variables to filter with
-        query.put(VariantDBAdaptor.REGION, "20:61000-66000,20:63000-69000");
-
-        VariantStorageManager variantStorageManager = StorageManagerFactory.getVariantStorageManager();
-        VariantDBAdaptor variantDBAdaptor = variantStorageManager.getDBAdaptor(DB_NAME, null);
-        VariantSourceDBAdaptor variantSourceDBAdaptor = variantDBAdaptor.getVariantSourceDBAdaptor();
-
-        VariantExporter variantExporter = new VariantExporter(cellBaseClient, variantDBAdaptor, query);
-        List<String> outputFiles = variantExporter.vcfExport(outputDir, variantSourceDBAdaptor);
-
-        ////////// checks
-
-        assertEquals(studies.size(), outputFiles.size());
-        assertEquals(0, variantExporter.getFailedVariants());   // test file should not have failed variants
-
-        VariantDBIterator iterator = variantDBAdaptor.iterator(query);
-        assertEqualLinesFilesAndDB(outputFiles, iterator);
-
-        for (String outputFile : outputFiles) {
-            assertVcfOrderedByCoordinate(outputFile);
-            boolean delete = new File(outputFile).delete();
-            assertTrue(delete);
-        }
+    public void testExportTwoStudies() throws Exception {
+        List<String> studies = Arrays.asList("7", "8");
+        String region = "20:71000-79000";
+        // TODO Both studies have 219 variants in the region... check if we can choose some region with different number of variants
+        QueryOptions query = new QueryOptions();
+        Map<String, List<VariantContext>> exportedVariants = exportAndCheck(query, studies, region);
+        checkExportedVariants(query, studies, exportedVariants);
     }
 
-    @Test
-    public void testMissingStudy() throws Exception {
-        List<String> files = Arrays.asList("5");
-        List<String> studies = Arrays.asList("7", "9"); // study 9 doesn't exist
-        QueryOptions query = getQuery(files, studies);
-        String outputDir = "/tmp/";
-
-        VariantStorageManager variantStorageManager = StorageManagerFactory.getVariantStorageManager();
-        VariantDBAdaptor variantDBAdaptor = variantStorageManager.getDBAdaptor(DB_NAME, null);
-        //VariantDBIterator iterator = variantDBAdaptor.iterator(query);
-        VariantSourceDBAdaptor variantSourceDBAdaptor = variantDBAdaptor.getVariantSourceDBAdaptor();
-
-        VariantExporter variantExporter = new VariantExporter(cellBaseClient, variantDBAdaptor, query);
+//    @Test
```

Comment:
Remove commented code block, all the test methods seem to have already been migrated to VariantExporterControllerTest.